### PR TITLE
[API_PARSER][CYBEREASON] Add support for malware URI

### PR DESCRIPTION
**Pull request by nlancon**

### CODE REVIEW

File: vulture_os/toolkit/api_parser/cybereason/cybereason.py [hunk 1]
```diff
@@ -321,49 +371,51 @@ def suspicionsNetwork(self, alertId):
...
+                domain = (user['displayName']).split('\\')[0]
+                if domain not in flattened_domain:
+                    flattened_domain.append(domain)
 
-        log['flattened_ip'] = flattened_ip
-        log['flattened_hostname'] = flattened_hostname
-        log['flattened_domain'] = flattened_domain
+            log['flattened_ip'] = flattened_ip
+            log['flattened_hostname'] = flattened_hostname
+            log['flattened_domain'] = flattened_domain
```
Comment (frikilax) [inline vulture_os/toolkit/api_parser/cybereason/cybereason.py:418]: `related` root object exists in ECS, is that taken into account?

File: vulture_os/toolkit/api_parser/cybereason/cybereason.py [hunk 2]
```diff
@@ -171,103 +174,150 @@ def test(self):
...
-    def addEnrichment(self, alert):
+    def get_malwares(self, since):
+        malware_uri = f"{self.host}/{self.MALWARE_URI}"
 
-        alert['id'] = alert['guid']
+        query = {
+            'filters': [{
+                'fieldName': 'timestamp',
+                'operator': 'GreaterThan',
+                'values': [int(since.timestamp()) * 1000]
```
[the remark this answers is not in the document]
Comment (nlancon) [reply vulture_os/toolkit/api_parser/cybereason/cybereason.py:194]: Yes, this function came from older version before JPO refacto and must run exactly like that

File: vulture_os/toolkit/api_parser/cybereason/cybereason.py (hunk 1 above)
Comment (nlancon) [reply vulture_os/toolkit/api_parser/cybereason/cybereason.py:418]: Yes, flattened fields is mapped to `related` ECS object

Comment (nlancon) [reply vulture_os/toolkit/api_parser/cybereason/cybereason.py:418]: Yes these fields are mapped to `related` ECS object

**Review by frikilax — APPROVED** [review]

**Pull request merged**